src/bootstrap/src/bin/main.rs

```diff
@@ -57,7 +57,9 @@ fn main() {
             }
             err => {
                 drop(err);
-                if let Ok(pid) = pid {
+                // #135972: We can reach this point when the lock has been taken,
+                // but the locker has not yet written its PID to the file
```

Comment:
Shouldn't this comment be in the 2nd branch arm?

Reply:
It's more to explain why _here_ we're checking both that the file was read and non-empty,  since there are a few reasons we can hit the second branch